tools/ci/solidity_coverage

```diff
@@ -3,4 +3,4 @@
 set -e
 
 yarn workspace @chainlink/contracts compile
-yarn workspace @chainlink/contracts coverage
+NODE_OPTIONS="--max-old-space-size=5120" yarn workspace @chainlink/contracts coverage
```

Comment:
was this only needed for the coverage analysic? or did we have the error on tests too?

Reply:
Also - can we move this env var to the workflow? That feels like a better place for it than in this script. `.github/workflows/solidity.yml`

Reply:
Only for coverage, I haven't seen any failures on the test side